jenkinsapi/plugins.py

```diff
@@ -44,6 +46,12 @@ def check_updates_server(self):
         )
         self.jenkins_obj.requester.post_and_confirm_status(url, params={}, data={})
 
+    @property
+    def update_center_dict(self):
+        update_center = 'https://updates.jenkins-ci.org/update-center.json'
```

Comment:
There is an API in Jenkins to get that information: http://localhost:8080/updateCenter/api/json?pretty=true&depth=1

As this is normal Jenkins REST API - it supports `tree` parameter, so it is possible to look up only required information: http://localhost:8080/updateCenter/api/json?pretty=true&depth=1&tree=availables[name,url,dependencies]

Also, with API there is no need to use json and all downloading/parsing to dict legwork will be handled by existing code in `get_data`. You only need to overridde `poll()` method, like it is done in other classes. 

Other classes cache their own data, so number of calls to Jenkins will be minimized. This call for plugins is very heavy and will not change between subsequent calls to `Plugins` class.

And a last thing - I think that it will be a bit friendly to log INFO line stating that this call may take considerable time. It takes seconds to execute such call, so, w/o warning of any kind, it may confuse users.